parts/xmem_nozzle.go

```diff
@@ -1104,54 +1103,42 @@ func (xmem *XmemNozzle) processData_sendbatch(finch chan bool, waitGrp *sync.Wai
 				goto done
 			}
 
-			if len(batch.getMetaSpec) == 0 {
-				// There is no getMetaSpec. This is the default code path with no CCR or mobile.
-				// We get meta to find what needs not be sent. If getMeta fails, we just send and let target
-				// perform CR.
-				noRepMap, err := xmem.batchGetMeta(batch.getMeta_map)
-				if err != nil {
-					xmem.Logger().Errorf("%v batchGetMeta failed. err=%v\n", xmem.Id(), err)
-				} else {
-					batch.noRep_map = noRepMap
+			// batchGet may use getMeta or subdoc_multi_lookup to get the results.
+			noRepMap, conflictMap, sendLookup_map, _, err := xmem.batchGet(batch.getMeta_map, batch.getMetaSpecWithHlv, batch.getMetaSpecWithoutHlv)
```

Comment:
Can't change the signature for batchGet since it is also called at line 1120 with different inputs.
in dataBatch, we have getMeta_map, noRep_map, mergeLookup_map, sendLookup_map. I will have to change all of them to be consistent. I will do that